03LessonThree/in-class-project/daily-planet.html

```diff
@@ -5,6 +5,33 @@
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
+    <main>
+      <header style="background-color:grey">
+      <h1> Daily Planet </h1>
+      <nav>
+    <a href="http://sports.yahoo.com/" target="_blank">Sports</a> |
+    <a href="https://weather.com/" target="_blank">Weather</a> |
+    <a href="http://www.huffingtonpost.com/section/politics" target="_blank">Politics</a> |
+    <a href="http://www.cnn.com/tech" target="http://www.cnn.com/tech">Technology</a>
+  </nav>
+</header>
+<section>
+  <article>
+    <h1 style="background-color:grey">When Zika Got Political,Americans Cities Got Screwed</h1>
+      <p> Author: Emma Grey Ellis</p>
+        <img src="https://www.wired.com/wp-content/uploads/2016/08/GettyImages-584696822-1024x683.jpg"
+<p> THE ZIKA VIRUS has officially arrived in the United States, as experts always knew it would. Florida Governor Rick Scott has announced that 15 people have contracted Zika from local mosquitoes in Miami. In response, the CDC put out a travel advisory warning pregnant women against traveling to the Zika-affected part of Miami—the first time such a warning has ever been applied to part of the continental US. And to combat the virus, Scott has requested a CDC Emergency Response Team to support the state’s research and blood sample collection.</p>
+  </section>
+  <aside>
+    <h4>Coca-Cola</h4>
+<a href="http://www.coca-cola.com/global/glp.html" target="_blank"><img src="https://upload.wikimedia.org/wikipedia/commons/2/2c/Flasche_Coca-Cola_0,2_Liter.jpg"/><a/>
+      <p>
+
+      </aside>
+</main>
+<footer>
+  <img src="https://upload.wikimedia.org/wikipedia/commons/4/42/Copyright_symbol.png"/>
+<img src="https://www.facebookbrand.com/img/fb-art.jpg"/>
+<img src="http://ericbrown.com/wp-content/uploads/2013/05/twitter-bird-blue-on-white.png"/>
     </body>
-</html>
\ No newline at end of file
+</html>

```

Comment:
Great work, Rodrigo! If you want to spend more time on layout, consider centering your header and footer content.